scss/civicrm/_variables.scss

```diff
@@ -1,4 +1,6 @@
+/* stylelint-disable max-nesting-depth, selector-max-compound-selectors, selector-no-qualifying-type, selector-max-id */
 $contact-avatar: 90px;
+$crm-control-height: 30px;
```

Comment:
you are using this variable only in one place, there is no need to define it (https://sass-guidelin.es/#variables)

Reply:
@AkA84 We are also using it on other places like https://github.com/civicrm/org.civicrm.shoreditch/pull/159/files#diff-717f96292f9fd95fa7164dfbced0bd66R140
https://github.com/civicrm/org.civicrm.shoreditch/pull/159/files#diff-717f96292f9fd95fa7164dfbced0bd66R287

There are around 6 instance where we are using this variable.